plugins/lsp/lsp-status.nix

```diff
@@ -0,0 +1,83 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "lsp-status";
+  originalName = "lsp-status.nvim";
+  url = "https://github.com/nvim-lua/lsp-status.nvim";
+  defaultPackage = pkgs.vimPlugins.lsp-status-nvim;
+  maintainers = [ helpers.maintainers.b3nb5n ];
+  settingsOptions = with lib; {
```

Comment:
`with lib;` can be applied to the whole file, i.e. just before the call to `mkNeovimPlugin` (fairly common practice in nixvim).